unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs

```diff
@@ -5,7 +5,7 @@
 
 public class ChatHUDController : IDisposable
 {
-    public const int MAX_CHAT_ENTRIES = 100;
+    public static int MAX_CHAT_ENTRIES = 100;
```

Comment:
```suggestion
    public static int MAX_CHAT_ENTRIES { internal set; get; } = 100;
```
If we are going to transform this into a variable, can we modify it to minimize any risk of changing it value by mistake?